README.md

```diff
@@ -16,7 +16,7 @@ Status](https://deps.rs/repo/github/ratatui-org/ratatui/status.svg?style=flat-sq
 [![Discord](https://img.shields.io/discord/1070692720437383208?label=discord&logo=discord&style=flat-square)](https://discord.gg/pMCEU9hNEj)
 
 <!-- See RELEASE.md for instructions on creating the demo gif --->
-![Demo of Ratatui](https://github.com/ratatui-org/ratatui/assets/24392180/93ab0e38-93e0-4ae0-a31b-91ae6c393185)
```

Comment:
I would rely on GitHub for hosting major files like this. I have projects from 4+ years ago that still has no broken links/images since everything is hosted on GitHub. On the other hand, we don't know if `vhs.charm.sh` will go offline tomorrow or not 🐻

Reply:
It would be nice if we can upload it to a comment on a gist on GitHub programmatically. That way we don’t bloat the git history of this project. Or a separate repo just for demos also works.

Reply:
I made a issue to track this request: https://github.com/ratatui-org/ratatui/issues/401